app/views/idv/gpo_verify/index.html.erb

```diff
@@ -11,7 +11,10 @@
 
 <% title t('titles.verify_profile') %>
 
-<%= render PageHeadingComponent.new.with_content(t('forms.verify_profile.title')) %>
+<%= render PageHeadingComponent.new.with_content(t('forms.verify_profile.welcome_back')) %>
+<%= sanitize(t('forms.verify_profile.welcome_back_description')) %>
```

Comment:
I think `sanitize` is a nice alternative to [`_html`](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) to be a bit more restrictive about the types of HTML tags we're expecting. Maybe we could be even more explicit about it, than to depend on the Rails' defaults?

```suggestion
<%= sanitize(t('forms.verify_profile.welcome_back_description'), tags: %[p strong]) %>
```